sdk/src/main/java/com/silanis/esl/sdk/CreditCard.java

```diff
@@ -0,0 +1,32 @@
+package com.silanis.esl.sdk;
+
+
+public class CreditCard {
+
+    private String cvv;
+    private CcExpiration expiration;
+    private String name;
+    private String number;
+    private String type;
+
+    public void setCvv(String value){ cvv = value; }
```

Comment:
Please don't name the parameter as `value`, but name it as the actual parameter name

Reply:
Fixed.